### PR TITLE
Add Stara credentials documentation

### PR DESCRIPTION
**Pull request by palomacione**

### CODE REVIEW

**Comment by palomacione** [comment]
[[Postman] Add Stara Credentials documentation](https://trello.com/c/LZadvxbL/364-postman-add-stara-credentials-documentation)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -1174,6 +1174,161 @@
...
+													},
+													{
+														"key": "Content-Type",
+														"value": "application/json",
+														"type": "text"
+													}
+												],
+												"body": {
+													"mode": "raw",
+													"raw": "{\n    \"apiKey\": \"{{stara_api_key}}\",\n    \"accessToken\": \"{{stara_access_token}}\",\n    \"accessTokenClient\": \"{{stara_access_token_client}}\",\n    \"refreshToken\": \"{{stara_refresh_token}}\"\n}",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1258]: We need to update the postman collection to include the new variables and also explain what they are :smile:

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Nice work! We just have to update the postman environment!

File: Leaf API.postman_collection.json (hunk 1 above)
Comment (palomacione) [reply Leaf API.postman_collection.json:1258]: c826cb7d8b7d1656098c1cb5c5b0fa67594a5180
I think the names are pretty descriptive, do you think it needs more explanation?

Comment (Vinniekun) [reply Leaf API.postman_collection.json:1258]: I suppose the explanation of variables is something like this:

image

Is this right?

**Review by Vinniekun — APPROVED** [review]

**Review by ricardosantacattarina — APPROVED** [review]

**Review by fabyan-alexander — APPROVED** [review]

**Pull request merged**